### PR TITLE
Fix code overflow

### LINKED ISSUE #6769: Provider config in Site Administration page breaks UI
- Gitea version (or commit ref): https://github.com/go-gitea/gitea/commit/81059a2567be0cd811ca10e3545282c9aa53276a
- Git version: Not relevant
- Operating system: MacOS
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

Kindly take a look at the screenshot, it is scrollable all the way to the right 

Screenshot 2019-04-26 at 19 26 18

### PR DESCRIPTION
**Pull request by xf-**
Fix #6769

format code like log setting and ensure it breaks on small screens
image

image

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Comment by lafriks** [comment]
Conflict

**Comment by xf-** [comment]
@lafriks you want to merge this next? A compressed - single line css will always cause conflicts. i will rebase it. Maybe set less to single line per selector - less conflicts with multiple css changes

**Comment by kolaente** [comment]
@xf- You need to run `make generate-stylesheets` to create a new compiled stylesheet with your changes. We always keep the comiled less in the repo.

**Comment by xf-** [comment]
@kolaente forgot it while rebasing al branches

File: public/less/_admin.less [hunk 1]
```diff
@@ -64,4 +64,9 @@
             margin-left: 5px;
         }
     }
+
+    code, pre {
```
Comment (lafriks) [inline public/less/_admin.less]: `pre` is only used why add also `code`?
```suggestion
    pre {
```

Comment (xf-) [reply public/less/_admin.less]: `<code>` is also used on admin page and this avoids the issue in other fields (e.g. mobile) or move all formatted stuff to pre

Comment (lafriks) [reply public/less/_admin.less]: wouldn't just styling `code` fix also this without changing tags from `code` to `pre`?

Comment (xf-) [reply public/less/_admin.less]: thats why i'm added code aswell

Comment (xf-) [reply public/less/_admin.less]: All lines with `JsonPrettyPrint` in config use `<pre>`. I want a consistent behavior/style on same page.

**Comment by xf-** [comment]
firce push was for the linter

**Review by lafriks — APPROVED** [review]

**Pull request merged**